server/utils/constants.js

```diff
@@ -260,6 +262,7 @@ const DEVICE_FEATURE_TYPES = {
   LIGHT: {
     BINARY: 'binary',
     BRIGHTNESS: 'brightness',
+    FADE_IN: 'fade-in',
```

Comment:
What if fade out? Should be propose a 'fade' with in / out options?

Reply:
Maybe fade out can be developped as part of another feature